plugins/woocommerce/src/Blocks/Skeleton/CartSkeleton.php

```diff
@@ -0,0 +1,87 @@
+<?php
```

Comment:
I haven't dived deeply into the implementation yet but I am curious as to why `CartSkeleton.php` is so separate and not part of the `Cart.php` class? It seems really specific to the block and having it separate feels like it creates some fragmentation. What is the benefit to your approach here?